app/Services/DepositService.php

```
@@ -0,0 +1,158 @@
+            throw new Exception('Unable to process deposit. Please try again.');
+        }
+    }
+
+    /**
+     * Calculate deposit fee based on gateway
+     *
+     * @param float $amount
+     * @param string $gateway
+     * @return float
+     */
+    protected function calculateDepositFee(float $amount, string $gateway): float
```

PHPMD (2.15.0)

Avoid unused parameters such as '$amount'. (Unused Code Rules)

(UnusedFormalParameter)